addons/web/static/src/views/kanban/kanban_column_quick_create.js

```diff
@@ -11,7 +11,7 @@ export class KanbanColumnQuickCreate extends Component {
         this.dialog = useService("dialog");
         this.root = useRef("root");
         this.state = useState({
-            columnTitle: "",
+            hasInputFocused: undefined,
```

Comment:
hasInputFocused uses the state and is set only from the template.
I took the liberty to do that so people wouldn't think we forgot an empty useState use.